packages/ui/src/i18n/dictionaries/authenticator/types.ts

```diff
@@ -17,7 +17,7 @@ type AuthenticatorText =
   | 'Dismiss alert'
   | 'Email'
   | 'Enter your code'
-  | 'Enter your email'
+  | 'Enter your Email'
```

Comment:
This type was wrong and enforces the wrong translation key actually used by the Authenticator code:
https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/dictionaries/authenticator/defaultTexts.ts#L30